ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -763,7 +765,15 @@ def osd_encryption_verification():
     osd_node_names = get_osds_per_node()
     for worker_node in osd_node_names:
         lsblk_cmd = f"oc debug node/{worker_node} -- chroot /host lsblk"
-        lsblk_out = run_cmd(lsblk_cmd)
+        # It happens from time to time that we see this error:
+        # error: unable to create the debug pod "node-name-internal-debug
+        # Hence we need to add some re-try logic here
+        lsblk_out = retry(
+            (CommandFailed),
+            tries=3,
+            delay=60,
+            text_in_exception="unable to create the debug pod",
+        )(run_cmd)(lsblk_cmd)
```

Comment:
Please use https://github.com/red-hat-storage/ocs-ci/blob/f69e03df9dde1f07f3f6cf1671c46ef50ab2fad6/ocs_ci/ocs/ocp.py#L174 instead of run_cmd

Reply:
This function has a bit different logic and it doesn't return any message in the commandFailed exception here:
https://github.com/red-hat-storage/ocs-ci/blob/f69e03df9dde1f07f3f6cf1671c46ef50ab2fad6/ocs_ci/ocs/ocp.py#L199

So the logic I need will not probably work here.

Also I am not author of this code, just trying to improve the logic in the case the debug pod will fail to be created from some reason and re-try.

Also as part of this fix I don't want to update this exec_od_debug_cmd function at the moment as this is suppose to unblock one of our production upgrade jobs ASAP for 4.11 executions.